src/parser/jobs/rpr/modules/Reaver.tsx

```diff
@@ -103,7 +103,7 @@ export class Reaver extends Analyser {
 			</Trans>,
 			tiers: SEVERITIES,
 			why: <Trans id ="rpr.reaver.suggestion.dropped.why">
-					You used lost <Plural value={this.droppedReavers} one="# stack" other="# stacks"/> over the course of the fight.
+					You used <Plural value={this.droppedReavers} one="# stack" other="# stacks"/> over the course of the fight.
```

Comment:
this should be "lost" right? or is that not what droppedReavers means?

Reply:
Reading this I thought maybe I was high or something when I wrote this but it turns out following the exact request was a bad idea and the user wanted the wrong word dropped, will resolve in a bit.